.bazelrc

```diff
@@ -1,5 +1,5 @@
 build --enable_platform_specific_config=true
-query --enable_platform_specific_config=true
+# query --enable_platform_specific_config=true
```

Comment:
any particular reason for this to be commented out?

Reply:
There's a bug in the bb cli where if there is no matching config for the platform, the query fails. We don't have linux-specific query configs, so it doesn't work. Doesn't belong in this PR, though.